packages/http/lib/auth.tsp

```diff
@@ -111,7 +111,7 @@ model ApiKeyAuth<Location extends ApiKeyLocation, Name extends string> {
  * @template Scopes The list of OAuth2 scopes, which are common for every flow from `Flows`. This list is combined with the scopes defined in specific OAuth2 flows.
  */
 @doc("")
-model OAuth2Auth<Flows extends OAuth2Flow[], Scopes extends valueof string[] = []> {
+model OAuth2Auth<Flows extends OAuth2Flow[], Scopes extends string[] = []> {
```

Comment:
Should we add `valueof` again, even after resolving the breaking change that would give warning to everybody. This might be better to do as a seperate PR that would change all our decorators to go to value

Reply:
This should be valueof I think. A follow-up PR seems good though.